src/drivers/localstorage.js

```diff
@@ -123,7 +124,7 @@
                     value = serializer.deserialize(value);
                 }
 
-                value = iterator(value, key.substring(keyPrefixLength), i + 1);
+                value = iterator(value, key.substring(keyPrefixLength), iterationNumber++);
```

Comment:
Why this change? You can see above we use `i` as a zero-index through the keys, so adding one to it then returning does the same thing as returning your 1-indexed variable.